src/app/components/shared/form/form.component.ts

```diff
@@ -31,31 +35,68 @@ export class FormComponent extends withUnsubscribe() implements OnInit {
   @Input() public submitLoading: boolean;
   @Input() public subTitle?: string;
   @Input() public title?: string;
+  /**
+   * Recaptcha seed, using the following rules:
+   * string: Seed value
+   * true: Will set seed
+   * false: Will not set seed
+   */
+  @Input() public seed?: string | boolean = false;
+  private loadingSeed: boolean;
```

Comment:
these value definitions feel a bit icky. Maybe

undefined - will not use recaptcha
"LOADING" - waitng for recpatcha seed
string:  - seed available

Actually, straight from the TS  handbook: https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#discriminating-unions

```
type RecaptchaLoadingState = {
  state: "loading";
};
type RecaptchaLoadedState = {
  state: "loaded";
  seed: string;
};
type RecaptchaState = RecaptchaLoadingState | RecaptchaLoadedState ;
```

Then type declartion is `RecaptchaState`

Reply:
suggested name: `recaptchaSeed`